tests/v2/validation/connection/workload.go

```diff
@@ -0,0 +1,22 @@
+package connection
+
+import (
+	steveV1 "github.com/rancher/shepherd/clients/rancher/v1"
+	"github.com/rancher/shepherd/extensions/workloads"
+	corev1 "k8s.io/api/core/v1"
+)
+
+const (
+	defaultNamespace = "default"
+)
+
+func createDeployment(steveclient *steveV1.Client, containersTemplate []corev1.Container, containerName string) (*steveV1.SteveAPIObject, error) {
```

Comment:
can we re use the create function from here? https://github.com/rancher/shepherd/blob/main/extensions/workloads/deployment/create.go#L17.

we can then remove this helper altogether.